### DISCUSSION TITLE
Can private repository be taken down for any reason?

### DISCUSSION THREAD

**Discussion by turb1te** [community/community#22709] [New to GitHub]
<p>Hi!</p>
<p>I like a repository that I think will be taken down soon by DMCA or something else. I cloned(not fork) that repository as private. Can my private repository be taken down also? And also another example. If I upload repository that was taken down(along with the forks) by DMCA a few months ago, can it be removed if it is private? Can making a private repo be taken down for any copyright or other reasons?</p>
<p>Thanks! Have a nice day!</p>

**Comment by yamiacat** [comment, marked as answer]
<p>Hi there! The GitHub DMCA policy applies to <strong>all</strong> repos, and you can learn more here:</p>
<p><a href="https://help.github.com/en/github/site-policy/dmca-takedown-policy" class="onebox" target="_blank" rel="nofollow noopener">https://help.github.com/en/github/site-policy/dmca-takedown-policy</a></p>

**Reply by arainst** [reply]
Please come back my repository take down.
https://raw.githubusercontent.com/arainst/tv/main/Pbobo

**Comment by arainst** [comment]
Please come back my repository. 
https://raw.githubusercontent.com/arainst/tv/main/Pbobo
